Parser/asdl_c.py

```diff
@@ -2274,10 +2274,11 @@ def generate_module_def(mod, metadata, f, internal_h):
         #include "pycore_ast.h"
         #include "pycore_ast_state.h"     // struct ast_state
         #include "pycore_ceval.h"         // _Py_EnterRecursiveCall
-        #include "pycore_lock.h"          // _PyOnceFlag
         #include "pycore_interp.h"        // _PyInterpreterState.ast
+        #include "pycore_lock.h"          // _PyOnceFlag
         #include "pycore_modsupport.h"    // _PyArg_NoPositional()
         #include "pycore_pystate.h"       // _PyInterpreterState_GET()
+        #include "pycore_runtime.h"       // _Py_ID()
```

Comment:
`_Py_ID` is defined in `pycore_global_strings.h`

Reply:
Oh, I see your comment on the issue... it needs access to `_PyRuntime`'s internals.

Reply:
Yeah, in practice, pycore_runtime.h should be used.